Chaincase.iOS/Tor/TorProcessManager.cs

```diff
@@ -117,12 +149,12 @@ public void StartTor(OnionManagerDelegate managerDelegate)
                 Logger.LogInfo("Starting Tor");
             }
 
-            // Wait long enough for Tor itself to have started. It's OK to wait for this
-            // because Tor is already trying to connect; this is just the part that polls for
-            // progress.
-            DispatchQueue.MainQueue.DispatchAfter(new DispatchTime(DispatchTime.Now, TimeSpan.FromSeconds(1)), () =>
-            {
-                if (!(TorController?.Connected ?? false))
+			await Task.Delay(1000);
```

Comment:
This await.Delay replaces   DispatchQueue.MainQueue.DispatchAfter

Reply:
this changes the behavior when StartTor returns cause it waits vs the queued dispatch items being started after a second

Reply:
also spaces have been turned to tabs! please use spaces